iOS/Controllers/Dashboard/CalendarOverviewViewController.swift

```diff
@@ -159,7 +161,9 @@ class CalendarOverviewViewController: UIViewController {
 }
 
 extension CalendarOverviewViewController: ViewHeightDataSource {
-    var height: CGFloat { return 200 }
+    var height: CGFloat {
+        return self.eventsOverview.frame.maxY + 32
```

Comment:
What is the 32 for? If for the padding, then please refer to `self.eventOverview.layoutMargins`.

Reply:
Great, that's definitely the thing i was looking for.